transbot/models/pygithub_extension.py

```diff
@@ -0,0 +1,65 @@
+try:
+    import github
+    import github.GithubObject.NotSet as NotSet
+except ImportError:
+    github = None
+    NotSet = None
+from hashlib import sha1
+import base64
+
+
+def __get_file_sha1(file_content):
+    return sha1("blob %s\0%s" % (len(file_content), file_content)).hexdigest()
+
+
+def put_file_contents(self, path, message, content, sha=None,
+                      ref=NotSet):
+    """
+    :calls: `PUT /repos/:owner/:repo/contents/:path
+      <http://developer.github.com/v3/repos/contents>`_
+    :param path: string
+    :param message: string
+    :param content: string
+    :param ref: string
+    :rtype: None
+    """
+    assert isinstance(path, (str, unicode)), path
+    assert isinstance(path, (str, unicode)), message
```

Comment:
Shouldn't this be `assert isinstance(message, (str, unicode)), message` ?

Looks like copy paste errors